controllers/scenario/scenario_adapter.go

```diff
@@ -149,30 +119,8 @@ func (a *Adapter) EnsureDeletedScenarioResourcesAreCleanedUp() (controller.Opera
 		return controller.ContinueProcessing()
 	}
 
-	environmentsForScenario, err := a.loader.GetAllEnvironmentsForScenario(a.client, a.context, a.scenario)
```

Comment:
Is this removal necessary for this PR? If not, it can be handled as part of [STONEINTG-815](https://issues.redhat.com/browse/STONEINTG-815).